src/foam/u2/Accordion.js

```diff
@@ -0,0 +1,247 @@
+                label: 'Action 2',
+                action: this.someOtherFunction
+              }
+            ]
+          })
+            .add('This is hidden content...');
+      `
+    },
+    {
+      class: 'String',
+      name: 'expandIconPosition',
+      view: {
+        class: 'foam.u2.view.ChoiceView',
+        choices: [ 'left', 'right' ]
+      },
+      value: 'right'
+    },
+    {
+      class: 'Boolean',
+      name: 'expanded',
+      value: true
+    }
+  ],
+
+  methods: [
+    function init() {
+
+      const self = this;
+
+      this
+        .addClass()
+        .enableClass('expanded', this.expanded$)
+        .start('div')
+          .addClass(this.myClass('toolbar'))
+          .on('click', this.onToggle)
+          .start('div')
+            .addClass(this.myClass('title'))
+            .callIf(this.expandIconPosition === 'left', function() {
+              this.start(
+                self.TOGGLE
+              ).addClass(self.myClass('toggle'))
+              .on('click', self.onToggle);
+            })
+            .callIf(this.icon, function() {
+              this.tag(self.icon);
+            })
+            .start('span')
+              .addClass(this.myClass('title'))
+              .add(this.title$)
+            .end()
+          .end()
+          .start()
+            .addClass(this.myClass('actions'))
+            .forEach(this.actions, function(action) {
+              const actionId = `${self.id}-${action.name}`;
+              this.start(self.ActionView, {
+                id: actionId,
+                action: action.action,
+                data: action.data || self,
+                icon: action.icon || '',
+                label: action.label
+              })
+              .addClass(self.myClass('action'))
+              .callIf(action.name, function() {
+                this.addClass(self.myClass(action.name));
+              })
+              .callIf(action.tooltip, function() {
+                this.tooltip = action.tooltip;
+              })
+              .callIf(action.hoverColor, function() {
+                const style = self.document.createElement('style');
+                style.id = `${actionId}-hover-style`;
+                style.textContent = `
+                  #${actionId}:hover {
+                    background-color: ${action.hoverColor} !important;
+                  }
+                `;
+                self.document.head.appendChild(style);
+              })
+            })
+            .callIf(this.expandIconPosition === 'right', function() {
+              this.start(
+                self.TOGGLE
+              ).addClass(self.myClass('toggle'))
+              .on('click', self.onToggle);
+            });
+
+      this.start('div', null, this.content$)
+        //.show(this.expanded$)
+        .addClass(this.myClass('content'))
+      .end();
+    }
+  ],
+
+  actions: [
+    {
+      name: 'toggle',
+      label: '',
+      icon: 'images/cheveron-right.svg',
```

Comment:
Can use:
```suggestion
      themeIcon: 'next',
```
It will render svgs instead of images. Might better quality and scaling